tests/tests/test_github_webhooks.py

```diff
@@ -0,0 +1,540 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import requests
+import time
+
+
+def test_github_webhook(integration_test_runner_url):
+    payload = """{
```

Comment:
Could we read this from a file? The file could be something like `tests/tests/data/open_pull_request.json`.

I think will make it for a more readable test,  easier to maintain (maybe GitHub adds fields, we just replace the files), and more encouraging to create new test cases following the same design in the future.